lsp/ty.lua

```diff
@@ -0,0 +1,24 @@
+---@brief
+---
+--- https://github.com/astral-sh/ty
+---
+--- `ty`, a static type checker and language server for python.
+---
+--- The language server can be installed with `uv`:
+--- ```sh
+--- uv tool install ty
+--- ```
```

Comment:
Let's instead link to the documentation: https://github.com/astral-sh/ty/blob/main/README.md#getting-started. Yes, for now the documentation is on GitHub :)